apps/wallet/browser/kernel/handlers/common.ts

```diff
@@ -1,16 +1,16 @@
-import type { ConnectOptions } from '@walless/core';
 import { Networks } from '@walless/core';
-import { ResponseCode } from '@walless/messaging';
+import { ResponseCode } from '@walless/core';
+import type { ConnectOptions } from '@walless/sdk';
 import type { PublicKeyDocument, TrustedDomainDocument } from '@walless/store';
 import { selectors } from '@walless/store';
+import { storage } from 'utils/storage/db';
```

Comment:
I was wondering if we import `utils/storage`in `browser`. For kernel only, I created a storage instance in `apps/wallet/browser/kernel/utils/storage.ts`. I think we should keep platforms or environments separately